File: AssetManagement.Application/Dashboard/GetStoreDashboard/StoreDashboardResponse.cs
namespace AssetManagement.Application.Dashboard.GetStoreDashboard
{
    public sealed record StoreDashboardResponse
    (
        Guid StoreId,
        string StoreNumber,
        string StoreName,
        int TotalPDTs,
        int UsedPDTs,
        int ServicePDTs,
        int AvailablePDTs
    );
}

File: Client/src/app/features/dashboard/data-access/models/store-dashboard-response.ts
export interface StoreDashboardResponse {
  readonly storeId: string;
  readonly storeNumber: string;
  readonly storeName: string;
  readonly totalPDTs: number;
  readonly usedPDTs: number;
  readonly servicePDTs: number;
  readonly availablePDTs: number;
}
